include/envoy/upstream/health_checker.h

```diff
@@ -38,5 +43,8 @@ class HealthChecker {
 
 typedef std::shared_ptr<HealthChecker> HealthCheckerSharedPtr;
 
+std::ostream& operator<<(std::ostream& out, HealthState state);
```

Comment:
Please make sure these print helpers have full test coverage if we are going to have them.

Reply:
Good point. Will do.